src/differenceWith.js

```diff
@@ -11,7 +11,7 @@ var _curry3 = require('./internal/_curry3');
  * @memberOf R
  * @since v0.1.0
  * @category Relation
- * @sig (a -> a -> Boolean) -> [*] -> [*] -> [*]
+ * @sig ((a, a) -> Boolean) -> [a] -> [a] -> [a]
```

Comment:
@buzzdecafe that was my assumption as well when I made the signatures `... -> [a] -> [a] -> [a]`.